src/main/java/org/folio/services/adjusment/AdjustmentsService.java

```diff
@@ -48,7 +48,7 @@ public List<Adjustment> getProratedAdjustments(InvoiceLine invoiceLine) {
     return filterAdjustments(invoiceLine.getAdjustments(), INVOICE_LINE_PRORATED_ADJUSTMENT_PREDICATE);
   }
 
-  private List<Adjustment> filterAdjustments(List<Adjustment> adjustments, Predicate<Adjustment> predicate) {
+  public static List<Adjustment> filterAdjustments(List<Adjustment> adjustments, Predicate<Adjustment> predicate) {
```

Comment:
AdjustmentsService is not utility class, so it is a bad practice to make method as static here.
You can create instance of this service using constructor

Reply:
yes,i agree.